trace-dispatcher/src/Cardano/Logging/Trace.hs

```diff
@@ -25,12 +25,13 @@ module Cardano.Logging.Trace (
   , withDetails
   , foldTraceM
   , foldMTraceM
+  , foldMCondTraceM
```

Comment:
Can be removed, as a combination of foldMTraceM with filter can do the same. See above

Reply:
Not really, explanation in the original question.